common/data_refinery_common/models/models.py

```diff
@@ -738,6 +749,16 @@ def get_display_name(self):
         else:
             return self.filename
 
+    def get_extension(self):
+        """ Returns the lowercased extension of the filename
+        Thanks to https://stackoverflow.com/a/541408/763705 """
+        return os.path.splitext(self.filename)[1].lower()
+
+    def is_blacklisted(self):
+        # TODO: extend this list.
```

Comment:
I think we can just remove this TODO now. This isn't something that's on our radar or that I've seen any need for.